backend/dal/Migrations/v01.02.00/Up/PostUp/01-ViewProperties.sql

```diff
@@ -0,0 +1,219 @@
+    , [Classification] = c.[Name]
+    , p.[AgencyId]
+    , [Agency] = ISNULL(pa.[Name], a.[Name])
+    , [AgencyCode] = ISNULL(pa.[Code], a.[Code])
+    , [SubAgency] = (
+        SELECT CASE WHEN pa.[Id] IS NOT NULL
+            THEN a.[Name]
+            ELSE NULL
+        END)
+    , [SubAgencyCode] = (
+        SELECT CASE WHEN pa.[Id] IS NOT NULL
+            THEN a.[Code]
+            ELSE NULL
+        END)
+    , p.[AddressId]
+    , [Address] = TRIM(ISNULL(adr.[Address1], '') + ' ' + ISNULL(adr.[Address2], ''))
+    , [AdministrativeArea] = adr.[AdministrativeArea]
+    , [Province] = ap.[Name]
+    , adr.[Postal]
+    , p.[ProjectNumber]
+    , p.[Name]
+    , p.[Description]
+    , p.[Location]
+    , p.[Boundary]
+    , p.[IsSensitive]
+    , p.[IsVisibleToOtherAgencies]
+
+    -- Parcel Properties
+    , p.[PID]
+    , p.[PIN]
+    , [LandArea] = p.[LandArea]
+    , [LandLegalDescription] = p.[LandLegalDescription]
+    , [Zoning] = p.[Zoning]
+    , [ZoningPotential] = p.[ZoningPotential]
+
+    -- Building Properties
+    , [ParcelId] = p.[Id]
+    , [BuildingConstructionTypeId] = 0
+    , [BuildingConstructionType] = null
+    , [BuildingFloorCount] = 0
+    , [BuildingPredominateUseId] = 0
+    , [BuildingPredominateUse] = null
+    , [BuildingOccupantTypeId] = 0
+    , [BuildingOccupantType] = null
+    , [BuildingTenancy] = null
+    , [RentableArea] = 0
+    , [LeaseExpiry] = null
+    , [OccupantName] = null
+    , [TransferLeaseOnSale] = CAST(0 AS BIT)
+
+    , [Assessed] = ISNULL(eas.[Value], 0)
+    , [AssessedDate] = eas.[Date]
+    , [Appraised] = ISNULL(eap.[Value], 0)
+    , [AppraisedDate] = eap.[Date]
+    , [Estimated] = ISNULL(fe.[Value], 0)
+    , [EstimatedFiscalYear] = fe.[FiscalYear]
+    , [NetBook] = ISNULL(fn.[Value], 0)
+    , [NetBookFiscalYear] = fn.[FiscalYear]
+FROM dbo.[Parcels] p
+JOIN dbo.[PropertyClassifications] c ON p.[ClassificationId] = c.[Id]
+JOIN dbo.[Agencies] a ON p.[AgencyId] = a.[Id]
+LEFT JOIN dbo.[Agencies] pa ON a.[ParentId] = pa.[Id]
+JOIN dbo.[Addresses] adr ON p.[AddressId] = adr.[Id]
+JOIN dbo.[Provinces] ap ON adr.[ProvinceId] = ap.[Id]
+OUTER APPLY (
+    SELECT TOP 1
+        [Value]
+        , [Date]
+    FROM dbo.[ParcelEvaluations]
+    WHERE [ParcelId] = p.[Id]
+        AND [Key] = 0 -- [Assessed]
+    ORDER BY [Date] DESC
+) AS eas
+OUTER APPLY (
+    SELECT TOP 1
+        [Value]
+        , [Date]
+    FROM dbo.[ParcelEvaluations]
+    WHERE [ParcelId] = p.[Id]
+        AND [Key] = 1 -- Appraised
+    ORDER BY [Date] DESC
+) AS eap
+OUTER APPLY (
+    SELECT TOP 1
+        [Value]
+        , [FiscalYear]
+    FROM dbo.[ParcelFiscals]
+    WHERE [ParcelId] = p.[Id]
+        AND [Key] = 1 -- Estimated
+    ORDER BY [FiscalYear] DESC
+) AS fe
+OUTER APPLY (
+    SELECT TOP 1
+        [Value]
+        , [FiscalYear]
+    FROM dbo.[ParcelFiscals]
+    WHERE [ParcelId] = p.[Id]
+        AND [Key] = 0 -- NetBook
+    ORDER BY [FiscalYear] DESC
+) AS fn
+UNION ALL
```

Comment:
Had to switch to a `UNION ALL` because spatial data types aren't allowed in an normal `UNION`.  I don't know if this makes this perform poorly or not.